rust/connlib/tunnel/src/dns.rs

```diff
@@ -291,6 +291,11 @@ impl StubResolver {
 
                 vec![AllRecordData::Ptr(domain::rdata::Ptr::new(fqdn))]
             }
+            (Rtype::HTTPS, Some(_)) => {
+                anyhow::bail!(
```

Comment:
And somewhere up the stack, this turns into an error log, not bailing out of the session, right?

Reply:
almost, `trace` log because it happens very often [here](https://github.com/firezone/firezone/blob/fix/connlib/blackhole-https-resources/rust/connlib/tunnel/src/client.rs#L661)